lib/matplotlib/patches.pyi

```diff
@@ -74,7 +74,7 @@ class Patch(artist.Artist):
 
 class Shadow(Patch):
     patch: Patch
-    def __init__(self, patch: Patch, ox: float, oy: float, **kwargs) -> None: ...
+    def __init__(self, patch: Patch, ox: float, oy: float, *, shade: float, **kwargs) -> None: ...
```

Comment:
```suggestion
    def __init__(self, patch: Patch, ox: float, oy: float, *, shade: float = …, **kwargs) -> None: ...
```
?

Reply:
That `…` doesn’t look right 😕

Reply:
I thought this was a comment about `...` being used for "there is a default" in the stub files.